cmd/example-client/main.go

```diff
@@ -118,9 +124,46 @@ func ModifyTarget(msg info.IDInfo, _ []byte) *connection.SendResponse {
 	return messages.EventToResponse(context, modify)
 }
 
+func GetVT(_ info.IDInfo, msg []byte) *connection.SendResponse {
+	var result models.Result
+	err := json.Unmarshal(msg, &result)
+	if err != nil {
+		log.Panicf("unable to parse result: %s", string(msg))
+	}
+	if result.OID == "" {
+		return nil
+	}
+
+	getVT := models.GetVT{
+		Identifier: messages.Identifier{
+			ID:      result.ScanId,
```

Comment:
What is the reasoning to use a scan id within get.vt instead of setting id to the oid and skip the extra field?